src/components/SearchForm/SearchForm.spec.tsx

```diff
@@ -0,0 +1,74 @@
+describe('SearchForm', () => {
```

Comment:
Please change the title to 'Search Form' instead of 'SearchForm'

Reply:
Done, changed to 'Search Form'